docs/release-notes/latest.md

```diff
@@ -4,174 +4,106 @@ append_help_link: true
 
+## 🔧 Semgrep OSS Engine
 
-### Changed
+* The following versions of Semgrep OSS Engine were released in September 2023:
+  * [<i class="fas fa-external-link fa-xs"></i> 1.38.0](https://github.com/returntocorp/semgrep/releases/tag/v1.38.0)
+  * [<i class="fas fa-external-link fa-xs"></i> 1.39.0](https://github.com/returntocorp/semgrep/releases/tag/v1.39.0)
+  * [<i class="fas fa-external-link fa-xs"></i> 1.40.0](https://github.com/returntocorp/semgrep/releases/tag/v1.40.0)
+  * [<i class="fas fa-external-link fa-xs"></i> 1.41.0](https://github.com/returntocorp/semgrep/releases/tag/v1.41.0)
+  * [<i class="fas fa-external-link fa-xs"></i> 1.42.0](https://github.com/returntocorp/semgrep/releases/tag/v1.42.0)
 
-- Running the `semgrep`  command with no subcommand now displays the help message. Previously, the `semgrep` command ran a SAST scan by default.
+## 🌐 Semgrep Cloud Platform
 
-### Removed
+### Added
 
-- `python -m semgrep` has been removed. Instead, invoke Semgrep directly by entering `semgrep` in the CLI.
-- Semgrep no longer looks for a `.semgrep.yml` config file or `.semgrep/` in the current directory, which previously caused conflicts when invoking `semgrep` from your home directory. This is because the home directory can contain a `.semgrep/settings.yml` file that is not a Semgrep rule, despite Semgrep expecting a rule file. ([#4457](https://github.com/returntocorp/semgrep/issues/4457))
-  - The preferred method to run rules is to explicitly pass rules through the `--config` option. For example, to run a `.semgrep.yml` file containing rules, you must enter `semgrep --config .semgrep.yml`.
-- If you previously wrapped Semgrep Python code by calling `semgrep_main.main`, you must replace the previous call with `run_scan.run_scan`. Note that these Python calls will be removed in the future.
-- `--enable-metrics` and `--disable-metrics` have been removed. Instead, use any of the following:
-  - `--metrics=on`
-  - `--metrics=off`
-  - `--metrics=auto`
+- UX: Added a new **onboarding** flow. This onboarding flow streamlines the following steps to ensure that users are able to quickly add repositories for scanning with Semgrep: <!-- #10473 -->
+	- **Deployment creation**. The Semgrep team has made improvements to Semgrep account creation and connecting your source code manager, such as GitHub or GitLab. 
+	- **Onboarding checklist.** This helps you troubleshoot and resolve any issues early on in your journey.
+	- **Tour of features**. Make the most of your Semgrep experience by learning what features are available to you.
+- **GitHub Enterprise:** Triage-by-PR is now enabled for public repositories. For this specific configuration, you may need to contact support@semgrep.com to enable this feature. <!-- #10143 -->
```

Comment:
I'm happy to advertise this but @sachishah thoughts on if you want to? We built this only really for yahoo